### PR TITLE
Force upgrade sticky ad to 1.0

### PR DESCRIPTION
**Pull request by zhouyx**
For #6169 
cc @jasti 
I will remove the actually file once the upgrade is stable in prod.

### CODE REVIEW

**Comment by dvoytenko** [comment]
Why are we updating only tests? How would the actual upgrade happen? Are you saying that you are deploying 1.0 to 0.1 instead?

**Comment by zhouyx** [comment]
Yes. Use the file `amp-sticky-ad-1.0.js` for sticky-ad 0.1 in build stage.

**Comment by dvoytenko** [comment]
LGTM, but please wait for @erwinmombay

**Comment by zhouyx** [comment]
ping @erwinmombay

File: gulpfile.js [hunk 1]
```diff
@@ -378,6 +378,10 @@ function buildExtension(name, version, hasCss, options, opt_extraGlobs) {
   options = options || {};
   options.extraGlobs = opt_extraGlobs;
   var path = 'extensions/' + name + '/' + version;
+  if (name == 'amp-sticky-ad' && version == '0.1') {
+    // sticky-ad upgrade, 0.1  is deprecated.
```
Comment (erwinmombay) [inline gulpfile.js]: please add explanation why we do this

File: gulpfile.js [hunk 2]
```diff
@@ -378,6 +378,12 @@ function buildExtension(name, version, hasCss, options, opt_extraGlobs) {
   options = options || {};
   options.extraGlobs = opt_extraGlobs;
   var path = 'extensions/' + name + '/' + version;
+  if (name == 'amp-sticky-ad' && version == '0.1') {
+    // Special case for sticky-ad upgrade.
+    // To deprecate 0.1, replace the build path so that amp-sticky-ad-0.1.js
+    // is built from extensions/amp-sticky-ad/1.0/amp-sticky-ad.js
```
Comment (erwinmombay) [inline gulpfile.js]: can you explain further "why" we want this.

Comment (zhouyx) [inline gulpfile.js]: how do you like the explanation now : )

**Pull request merged**